services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -100,13 +102,35 @@ def endpoint(self) -> str:
         return self.S3_ENDPOINT
 
 
+class StorageSettings(BaseCustomSettings):
+    STORAGE_HOST: str = "storage"
+    STORAGE_PORT: int = 8080
+    STORAGE_VTAG: str = "v0"
+
+    @cached_property
+    def endpoint(self) -> str:
+        return AnyHttpUrl.build(
+            scheme="http",
+            host=self.STORAGE_HOST,
+            port=f"{self.STORAGE_PORT}",
+            path=f"/{self.STORAGE_VTAG}",
+        )
+
+    @cached_property
+    def storage_endpoint(self) -> str:
+        """used to re-create STORAGE_ENDPOINT: used by node_ports and must be
+        in style host:port
+        without scheme or version tag"""
+        return f"{self.STORAGE_HOST}:{self.STORAGE_PORT}"
```

Comment:
You might already be aware of this one, just wanted to point it out.

https://github.com/ITISFoundation/osparc-simcore/blob/122703f6cbd964701143d085f3b7a4203e33ec15/packages/settings-library/src/settings_library/utils_service.py#L23-L38

Reply:
nope, but good to know. anyway for this one it's a different one, and I expect it to disappear soonish